rust/tests/s3_test.rs

```diff
@@ -11,15 +11,12 @@ mod s3 {
     use deltalake::StorageError;
 
     fn setup() {
-        std::env::set_var("AWS_REGION", "us-west-2");
-        std::env::set_var("AWS_ACCESS_KEY_ID", "AKIAX7EGEQ7FT6CLQGWH");
-        std::env::set_var(
-            "AWS_SECRET_ACCESS_KEY",
-            "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",
-        );
+        std::env::set_var("AWS_REGION", "us-east-2");
```

Comment:
WDYT about getting the environment variables instead of setting them again, just for having one single source of truth 😄

Reply:
do you suggest something specific? I've just added a new one to already defined ones :)

Reply:
@fvaleye did you mean setting the environment variables once before we run any test?

Reply:
It's not something that rust/cargo supports out of the box if I remember correctly, it's probably best to tackle it as a separate task. I have filed an issue at https://github.com/delta-io/delta-rs/issues/176.